qiita_ware/test/test_commands.py

```diff
@@ -56,26 +56,31 @@ def test_list_scp_wrong_key(self):
             list_remote('scp://runner@localhost:'+self.remote_dir_path,
                         self.test_wrong_key)
 
-    def test_list_scp_nonexist_key(self):
+    def test_download_remote_nonexist_key(self):
```

Comment:
Not clear to me why this switched from a test of `list_remote` to a test of `download_remote` ... but if it is now for `download_remote`, might be nice to move it down next to the other test for `download_remote` (even though that one is currently commented out).